web/src/views/LoginPortal/ConsentView/ConsentView.tsx

```diff
@@ -50,8 +49,9 @@ const ConsentView = function (props: Props) {
     const styles = useStyles();
     const { t: translate } = useTranslation();
     const navigate = useNavigate();
+    const [searchParams] = useSearchParams();
     const redirect = useRedirector();
-    const consentID = useConsentID();
+    const consentID = searchParams.get("id");
```

Comment:
```suggestion
    const consentID = searchParams.get(id);
```

Is it worth making this a const?